Code/V2/src/components/key.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+import theme from '../themes/index';
+
+const Key = (props) => {
+        var keyColor; 
```

Comment:
We can simplify this logic of defining keyColor to:

const keyColor = theme.key[props.status];

Reply:
Additionally, whenever you can write something as "const" rather than "var" or "let", we should opt to use "const".